ffi/wasm/src/jwt.rs

```diff
@@ -1,3 +1,9 @@
+// We use #[deprecated] for `JwsAlg::ED22519` to tell users that this algorithm is invalid
+// (`JwsAlg::EdDSA` should be used instead) but we should keep it inside enum to allow decoding of
+// invalid tokens generated by other libraries such as `golang-jws`. However, to avoid compilation
+// warnings, we need to allow deprecated items inside this module.
+#![allow(deprecated)]
+
```

Comment:
(Same observation)

Reply:
Ah well... trick with the internal module will not work here, it will mess up generated wasm module, so I kept this attribute and changed comment to clarify why it is needed

Reply:
Thank you!